tests/test_regressions.py

```diff
@@ -960,3 +971,187 @@ def my_function(foobar, info=None):
 
         assert not _exit.called
         assert file_ir._file_ir == expected_file_ir
+
+
+class TestGeneratorAndComprehensionsAreDeeplyVisited:
+    def test_generator_expression(
```

Comment:
**question**: Not sure if the are any cases for ternary operators in comprehensions/generator?

So like `any(foo if foo else "not_foo" for foo in bar)`. Similar question for walrus operators? Like `any(sum(baz) for in bar if baz := foo["spam"])`